javascript/package.json

```diff
@@ -13,15 +13,15 @@
     },
     "repository": {
         "type": "git",
-        "url": "git+https://github.com/anag004/piranhaJS.git"
+        "url": "git+https://github.com/uber/piranha.git"
     },
     "keywords": [],
     "author": "",
     "license": "ISC",
     "bugs": {
-        "url": "https://github.com/anag004/piranhaJS/issues"
+        "url": "https://github.com/uber/piranha/issues"
     },
-    "homepage": "https://github.com/anag004/piranhaJS#readme",
+    "homepage": "https://github.com/uber/piranha#readme",
```

Comment:
Should this one be the global Piranha README or https://github.com/uber/piranha/blob/master/javascript/README.md ?

One can argue either way, and I am fine with going to top-level, specially for a link called `homepage`, but I thought I'd bring up the choice :)

Reply:
Since JS README was linked from homepage, I used that. Changed it now.